lib/puppet/util/ldap/connection.rb

```diff
@@ -66,6 +66,6 @@ def start
       @connection.set_option(LDAP::LDAP_OPT_REFERRALS, LDAP::LDAP_OPT_ON)
       @connection.simple_bind(user, password)
   rescue => detail
-      raise Puppet::Error, "Could not connect to LDAP: #{detail}", detail.backtrace
+      raise Puppet::Error, _("Could not connect to LDAP: #{detail}"), detail.backtrace
```

Comment:
I think I may need to redo it for the `util` directory. I think I used a different invocation that caused some stuff to get missed, due to there being a couple of non-Ruby files in this directory.